index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
```

Comment:
Please undo all changes to this file as they don't serve any purpose and generate noise.

Reply:
I will keep the third (empty) line for better visualization. The other line linking the stylesheet was removed.